scss/chart.scss

```diff
@@ -70,6 +70,11 @@ body {
 }
 
 .basemap-satellite, .basemap-dark, .basemap-odyssey, .basemap-decimal {
+    .gradient-legend .block .text  {
+        color: #eeeeee;
+        background: rgba(0,0,0,0.1);
```

Comment:
Why even 0.1 alpha? Could this not just be transparent?

Reply:
@jrajav I think this looks better on the light areas of the dark maps.
I'll open a PR in immerse to test thso